src/main/resources/templates/progress/teamList.html

```diff
@@ -17,12 +17,13 @@ <h2> Všechny týmy již zahájily hru!</h2>
             </div>
         </div>
         <h2>Stav hry</h2>
-        <div class="row">
+        <div class="row" >
             <div class="col-sm-12">
-                <p th:text="'Týmy se nacházejí na stanovištích: ' + ${minMaxStages.getMin()} + '-' + ${minMaxStages.getMax()}"></p>
+                <p th:if="${minMaxStages == null}">Všechny týmy již mají dokončeno!</p>
+                <p th:unless="${minMaxStages == null}" th:text="'Týmy se nacházejí na stanovištích: ' + ${minMaxStages.getMin()} + '-' + ${minMaxStages.getMax()}"></p>
             </div>
         </div>
-        <div class="row" th:if="${teamsFinished gt 0}">
+        <div class="row" th:if="${teamsFinished gt 0}" th:unless="${minMaxStages == null}">
```

Comment:
Asi už jsem brain dead. Ale nešlo by to spojit do jedné if podmínky?
jako aby tam nebylo if a unless zároveň? Docela jsem nad tím musel přemýšlet, než jsem si byl jistý, co to vlastně bude dělat :D